.bazelrc

```diff
@@ -1,19 +1,39 @@
+# Define a set up flag aliases, so people can use `--cross` instead of the
+# longer `//build/toolchains:cross_flag`.
+build --flag_alias=crdb_test=//build/toolchains:crdb_test_flag
+build --flag_alias=cross=//build/toolchains:cross_flag
+build --flag_alias=dev=//build/toolchains:dev_flag
+build --flag_alias=lintonbuild=//build/toolchains:nogo_flag
+build --flag_alias=nolintonbuild=//build/toolchains:nonogo_explicit_flag
+build --flag_alias=with_ui=//pkg/ui:with_ui_flag
+
+build:cross --cross
+build:dev --dev
+build:lintonbuild --lintonbuild
+build:nolintonbuild --nolintonbuild
+# Note: nonogo is classically the name of the nolintonbuild configuration.
```

Comment:
So what you're saying here is that the lintonbuild configuration should be called gonogo? ;D

Reply:
![image](https://user-images.githubusercontent.com/3423966/160916611-7817f318-c317-4468-98ef-8fd664a1b6f2.png)